docs/content/vault_eso/vault_cli_login.md

```diff
@@ -0,0 +1,39 @@
+# Authenticate with the vault CLI using k8s
+
+These are instructions on how you can login to our vault instance via the vault cli tool. This might be useful for creating secrets dynamically during a `CronJob` or a `Tekton Task`.
+
+## Pre-requisities
+* Install [vault CLI][1] and [jq][4]
+* Cluster should be [integrated with opf vault instance][2]
+* Make sure you [enable your namespace to fetch secrets from vault][3]
+* Know your cluster, env and namespace (referred to as `${cluster}`, `${env}` and `${namespace}` in this doc)
+
+## Steps
+
+Login a cluster your namespace is in with `oc login`.
+
+Get the SA token with `oc sa get-token vault-secret-fetcher -n ${namespace}` and save it.
+
+Set the vault instance address to our instance by exporting the `VAULT_ADDR` enviromental variable:
+
+
+```bash
+$ export VAULT_ADDR=https://vault-ui-vault.apps.smaug.na.operate-first.cloud
+```
+
+Get client token with vault cli using the following command:
+
+```bash
+$ vault write auth/${cluster}-k8s/login role=${env}-ops jwt=${SA_TOKEN} -format=json | jq -r '.auth.client_token'
```

Comment:
this seems to create a client token to be used later.
is there also a way to read/extract the client token?
Otherwise it would be good to mention that this is the only time you see the token, so store it in a safe place :)

Reply:
Yea this is the only time you see the token unless you generate a new on, I'll make sure to mention that in the docs.